img/icons/coal-f.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What do the trailing `-f`s mean in this svg file naming system?
- `coal-f.svg`

Reply:
We have other instances of these icons that aren't easily repeatable. We could impose a `fill` value for color. Another one for @master12. The `-f` is somewhat arbitrary.

Reply:
@jenniferthibault This one might move into another sprint, but we'll definitely come back to it.